cmd/zed/relationship.go

```diff
@@ -315,6 +317,28 @@ func writeRelationshipCmdFunc(operation v1.RelationshipUpdate_Operation) func(cm
 			return err
 		}
 
+		var contextualizedCaveat *v1.ContextualizedCaveat
+
+		caveatString := cobrautil.MustGetString(cmd, "caveat")
```

Comment:
Can there be multiple instances of the caveat flag? e.g. `--caveat x:y --caveat a:b`

Reply:
No

Reply:
Should this flag then be `caveat-context`?

Reply:
Its both the caveat and its context

Reply:
ok, nvm I was just confused. I realize this is on write.